zeebe/gateway-grpc/src/main/java/io/camunda/zeebe/gateway/interceptors/impl/AuthenticationInterceptor.java

```diff
@@ -71,6 +99,21 @@ public <ReqT, RespT> Listener<ReqT> interceptCall(
     }
   }
 
+  private String getUsernameFromClaims(final Map<String, Object> claims) {
+    final var username = claims.get("preferred_username");
```

Comment:
❓ Where does this `preferred_username` come from?

Reply:
This'll have to go as we don't know what claims are on a user's token. See comment below this as well.